New package: urxvt-perls-2.2

nem-arch opened the pull request.

dominikh commented:
1. One PR per independent package please
2. Personally I don't think we want a uzbl package. It still has no real releases, we don't want to add unnecessary git-based packages, and uzbl's recent development has been rather "slow" (months of no activity whatsoever.)
3. Same goes for urxvt-perls, ask them to make a new release and use that, instead of pulling from git.

nem-arch commented:
1) fixed, sorry.
2) there were commits in the next branch, which is the default. but i agree that dev is slow and asked them on a release.
3) did that, fixed it.

dominikh commented:
- the README.md should be installed with vdoc
- noarch=yes should be set. I'll do both changes myself to avoid extra roundtrips.

dominikh commented:
Merged as 47b29cf0298c7ff5e2ad06b0c50786fe89d3c30d
